lightning/src/ln/channel.rs

```diff
@@ -3537,112 +3647,32 @@ impl<Signer: Sign> Channel<Signer> {
 			}
 		});
 
-		if self.funding_tx_confirmations > 0 {
-			self.funding_tx_confirmations += 1;
+		self.update_time_counter = cmp::max(self.update_time_counter, highest_header_time);
```

Comment:
Don't we have a risk of breaking bolt 7 requirement by setting `update_time_counter` to highest header time  ? "MUST set timestamp to greater than 0, AND to greater than any previously-sent channel_update for this short_channel_id."

Reply:
its a `cmp::max` so it should never go down? But we definitely need to do this because many clients filter updates based on the timestamp, so we need the time to be at least as of very recently (without having a "real" time source).